src/ol/layer/BaseTile.js

```diff
@@ -150,11 +150,13 @@ class BaseTileLayer extends Layer {
    * });
    * ```
    * @param {import("../pixel").Pixel} pixel Pixel.
+   * @param {boolean} [opt_premultiplied=true] If `false` return the RGBA data for image tiles before
+   * alpha premultiplication (when used frequently this may incur a significant performance loss).
```

Comment:
If we are going to support this, we should accept an `options` object.  The non-api methods don’t need to change.